src/components/Viewport.ts

```diff
@@ -995,19 +992,25 @@ export default class Viewport {
       panels.forEach(panel => {
         const overlappedClass = panel.getOverlappedClass(equalSizeClasses);
         if (overlappedClass && !cached[overlappedClass]) {
-          fragment.appendChild(panel.getElement());
+          if (!options.renderExternal) {
```

Comment:
Omission of checking `renderExternal` was a bug?

Reply:
Yes, that made `flicking.resize()` not to work.